_includes/manuals/1.24/1.2_release_notes.md

```diff
@@ -1,6 +1,9 @@
 ## Release notes for {{page.version}}
 
 ### Headline features
+#### Azure Resource Manager support for Compute Resource
+Foreman can now provision it's hosts using Azure Resource Manager (AzureRM) cloud provider for Compute Resource. As part of credentials, it requires the tenant-id, client-id, client-secret and subscription-id based on the user's subscription to create the Compute Resource. Provisioning can be successfully done on Linux platforms with either Finish template or Userdata template. Currently, it supports single OS disk for the VM and uses public images provided by Azure Marketplace.
```

Comment:
Parameters for credentials seem to be a part of the implementation details, else everything looks good. 

What do you think @apuntamb?

Reply:
Makes sense. I'll cut that from here.

Reply:
Nit:  Foreman can now provision it's hosts using Azure Resource Manager (AzureRM) cloud provider `as` Compute Resource.

Curious if this sounds right? @apuntamb

Reply:
Or may be just `using Azure Resource Manager(AzureRM) as a Compute Resource` ?
Makes sense?

Reply:
done.